templates/zerver/find_account.html

```diff
@@ -38,12 +38,21 @@ <h1 class="get-started">{{ _("Find your Zulip accounts") }}</h1>
             {% else %}
             <div class="find-account-form">
                 <p>
-                    {% trans %}
-                    Enter your email address to receive an email with the URLs
-                    for all the Zulip Cloud organizations in which you have
-                    active accounts. If you have also forgotten your password,
-                    you can <a href="/help/change-your-password">reset it</a>.
-                    {% endtrans %}
+                    {% if corporate_enabled %}
+                        {% trans %}
+                        Enter your email address to receive an email with the URLs
+                        for all the Zulip Cloud organizations in which you have
+                        active accounts. If you have also forgotten your password,
+                        you can <a href="/help/change-your-password">reset it</a>.
+                        {% endtrans %}
+                    {% else %}
+                        {% trans %}
+                        Enter your email address to receive an email with the URLs
+                        for all the Zulip organizations on this server in which you have
+                        active accounts. If you have also forgotten your password,
+                        you can <a href="/help/change-your-password">reset it</a>.
+                        {% endtrans %}
+                    {% endif %}
```

Comment:
Does it make sense to have each sentence be its own translation block? I The second sentence is identical in the two copies, and I think that would save translators wasted work.